packages/@aws-cdk/aws-events/test/test.input.ts

```diff
@@ -53,7 +53,85 @@ export = {
                 'Fn::Join': [
                   '',
                   [
-                    '{"data":<f1>,"stackName":"',
+                    '{"data":"<f1>","stackName":"',
```

Comment:
I don't understand how this is okay to do. What if the data isn't a string though?

Reply:
Good question, I am not finding documentation non-strings. I'll check if the lone variable case (previously supported) needs to be quote free to support non-string primitives (numbers, bools, and objects).

Reply:
Confirmed with EB, non-strings need to have no quotes. Will take a look at this tonight and add test cases for non-strings.

Reply:
It seems that (from your own examples, and ❤️ thank you so much for updating the PR description so clearly):

```
{
   data: <myValue>, // OK
}
```

Is okay both for strings and non-strings, so that seems like the safer default behavior.

Isn't "not adding quotes anywhere" the right thing to do?

Reply:
Yep, new revision coming that maintains no quotes for simple values (so no tests need to be changed, yay). Gitpod went down again before I could update the PR last night :-).